consensus/consensus-types/src/order_vote_msg.rs

```diff
@@ -52,12 +52,6 @@ impl OrderVoteMsg {
         self.order_vote
             .verify(validator)
             .context("[OrderVoteMsg] OrderVote verification failed")?;
-
```

Comment:
Can you change the name of this function from `verify` to `verify_order_vote` to make it clear that we not verifying the QC here? Also, add a comment on why QC verification is not being done here.

Reply:
Done.